subgraph/origin/generated/schema.ts

```diff
@@ -0,0 +1,316 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
schema.ts needs to be generated again after merge of PR #326

Reply:
Yes. Understood.